### ISSUE TITLE
Default Date User Search

### ISSUE THREAD

**Issue by Vansevenant** [meeting-room-booking-system/mrbs-code#3514]
Hello,

first of all, MRBS is a fantastic project – many thanks!

We recently updated to version 1.11.3.

I hadn't noticed before, but when a user logs in and clicks on their name to view their bookings, there is no "from" date displayed. The field remains empty, it is only possible to select a date using the calendar. The username is automatically displayed as the name. On the homepage, the search works as usual with the automatically inserted start date. It also works in the report, just not in this specific place. Is this the intended behavior? If so, is it possible to change it for our environment?

Thank you very much and best regards.g

**Comment by campbell-m** [comment]
No, it's not the intended behaviour and I can't reproduce it.  Are you seeing any errors in your browser console (open Developer Tools)?

**Comment by Vansevenant** [comment]
Thank you for your fast response. Unfortunately, I couldn't find any clues or differences in the developer tools when invoking the search. But it is great to hear from you that is an intended behaviour. Can I provide you with more information, or do you perhaps know of another debugging trick?

**Comment by campbell-m** [comment]
Have a look in your PHP error log to see if there are any error messages there.  If you can't find your log set `$debug = true;` in your MRBS config file - but don't leave it like that for long in a production environment.

**Comment by Vansevenant** [comment]
Good advice, thanks. Unfortunately, I didn't see anything negative there either. The views look the same. I set up a completely new MRBS 1.11.3 environment with my existing config.inc.php, and the problem persists. I'll test the old environment later.

**Comment by Vansevenant** [comment]
In version 1.11.0, it worked for me, but after that, we encountered the issue. I found that if I comment out the line in the file search.php: 'htmlspecialchars(utf8_strftime($strftime_format['date_short'], $search_start_time));' and replace it with the line from version 1.11.0: 'htmlspecialchars(datetime_format($datetime_formats['date_search'], $search_start_time));,' it works again. Apparently, it has no impact on other pages; everything is displayed and functions correctly.

**Comment by campbell-m** [comment]
Ah, OK, I see what you mean now.  I had been testing as an admin and so the user name link was going to the Report page and not the Search page.  I'm not sure why your change made a difference: it didn't when I tested it.

The bug was another unintended consequence of Issue #66.  I have now fixed it in [38a7906](https://github.com/meeting-room-booking-system/mrbs-code/commit/38a7906348638ea25d9f1a13b1f307d7d43e0457).
